vispy/scene/cameras/perspective.py

```diff
@@ -30,12 +30,11 @@ class PerspectiveCamera(BaseCamera):
     _state_props = ('scale_factor', 'center', 'fov')
 
     def __init__(self, fov=60.0, scale_factor=None, center=None, **kwargs):
-        super(PerspectiveCamera, self).__init__(**kwargs)
+        super(PerspectiveCamera, self).__init__(fov=fov, **kwargs)
         # Camera transform
         self.transform = MatrixTransform()
 
         # Set camera attributes
-        self.fov = fov
```

Comment:
This will also need to be reverted as well as the code just above it which passes `fov` to the base camera *and* the changes in the base camera that set `self._fov = fov`. That attribute in the base camera should be set to `0.0` as before as an initialization step. The subclasses are then responsible for setting `self.fov` based on the user-provided kwarg which needs the checks that are in the `self.fov` property setter.

Reply:
@petaflot This line still needs to be put back, actually...everything from the above comment doesn't seem like it made it in after your merge.